istioctl/cmd/root.go

```diff
@@ -204,7 +203,6 @@ debug and diagnose their Istio mesh.
 	experimentalCmd.AddCommand(wait.Cmd(ctx))
 	experimentalCmd.AddCommand(config.Cmd())
 	experimentalCmd.AddCommand(workload.Cmd(ctx))
-	experimentalCmd.AddCommand(revision.Cmd(ctx))
```

Comment:
Do we need a release note for this? It's experimental, so I'm not sure.. I know we typically document its promotion and then removal later.